Parsers/ASimDns/Tests/Fortinet_FortiGate_Dns_SchemaTest.csv

```diff
@@ -0,0 +1,92 @@
+"(2) Info: Missing optional field [DnsResponseIpLongitude]"
+"(2) Info: Missing optional field [DnsResponseIpRegion]"
+"(2) Info: Missing optional field [DnsSessionId]"
+"(2) Info: Missing optional field [DstDescription]"
+"(2) Info: Missing optional field [DstDeviceType]"
+"(2) Info: Missing optional field [DstDomain]"
+"(2) Info: Missing optional field [DstDvcId]"
+"(2) Info: Missing optional field [DstDvcScopeId]"
+"(2) Info: Missing optional field [DstDvcScope]"
+"(2) Info: Missing optional field [DstFQDN]"
+"(2) Info: Missing optional field [DstGeoCity]"
+"(2) Info: Missing optional field [DstGeoLatitude]"
+"(2) Info: Missing optional field [DstGeoLongitude]"
+"(2) Info: Missing optional field [DstGeoRegion]"
+"(2) Info: Missing optional field [DstHostname]"
+"(2) Info: Missing optional field [DstOriginalRiskLevel]"
+"(2) Info: Missing optional field [DstRiskLevel]"
+"(2) Info: Missing optional field [DvcDescription]"
+"(2) Info: Missing optional field [DvcId]"
+"(2) Info: Missing optional field [DvcInterface]"
+"(2) Info: Missing optional field [DvcMacAddr]"
+"(2) Info: Missing optional field [DvcOriginalAction]"
+"(2) Info: Missing optional field [DvcOsVersion]"
+"(2) Info: Missing optional field [DvcScopeId]"
+"(2) Info: Missing optional field [DvcScope]"
+"(2) Info: Missing optional field [EventOwner]"
+"(2) Info: Missing optional field [EventReportUrl]"
+"(2) Info: Missing optional field [NetworkProtocolVersion]"
+"(2) Info: Missing optional field [RuleName]"
+"(2) Info: Missing optional field [RuleNumber]"
+"(2) Info: Missing optional field [Rule]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [SrcDeviceType]"
+"(2) Info: Missing optional field [SrcDvcId]"
+"(2) Info: Missing optional field [SrcDvcScopeId]"
+"(2) Info: Missing optional field [SrcDvcScope]"
+"(2) Info: Missing optional field [SrcFQDN]"
+"(2) Info: Missing optional field [SrcGeoCity]"
+"(2) Info: Missing optional field [SrcGeoLatitude]"
+"(2) Info: Missing optional field [SrcGeoLongitude]"
+"(2) Info: Missing optional field [SrcGeoRegion]"
+"(2) Info: Missing optional field [SrcOriginalRiskLevel]"
+"(2) Info: Missing optional field [SrcOriginalUserType]"
+"(2) Info: Missing optional field [SrcProcessGuid]"
+"(2) Info: Missing optional field [SrcProcessId]"
+"(2) Info: Missing optional field [SrcProcessName]"
+"(2) Info: Missing optional field [SrcRiskLevel]"
+"(2) Info: Missing optional field [SrcUserAWSId]"
+"(2) Info: Missing optional field [SrcUserAadId]"
+"(2) Info: Missing optional field [SrcUserId]"
+"(2) Info: Missing optional field [SrcUserOktaId]"
+"(2) Info: Missing optional field [SrcUserScopeId]"
+"(2) Info: Missing optional field [SrcUserScope]"
+"(2) Info: Missing optional field [SrcUserSessionId]"
+"(2) Info: Missing optional field [SrcUserSid]"
+"(2) Info: Missing optional field [SrcUserUid]"
+"(2) Info: Missing optional field [TenantId]"
+"(2) Info: Missing optional field [ThreatCategory]"
```

Comment:
can we set ThreatCategory like Botnet for events like 54600 and 54601?

Reply:
Added ThreatCategory, ThreatField and ThreatIpAddr